### DISCUSSION TITLE
Is there a plan to internationalize the project?

### DISCUSSION THREAD

**Discussion by xiexin12138** [FlowiseAI/Flowise#487] [Q&A]
During my use of Flowise, I find it very useful. I would like to inquire if there are plans for internationalization. Alternatively, can I contribute to the project by adding internationalization features?

**Comment by HenryHengZJ** [comment]
definitely! what kind of feature you are thinking of?

**Reply by xiexin12138** [reply]
The feature I want to add is actually quite common, which is a language switch selector that allows changing the language of relevant content in the project through a language selector.

**Reply by HenryHengZJ** [reply]
Oh that's cool! Feel free to take a stab 😊

**Reply by xiexin12138** [reply]
Hahaha, alright. I will design it first and then submit it according to the PR rules.
